petrock/vision.py

```diff
@@ -64,38 +52,26 @@ def send_image_to_moondream(self, base64_image):
                 {
                     "role": "user",
                     "content": [
-                        {"type": "image_url",
-                         "image_url": {
-                            "url": f"data:image/jpeg;base64,{base64_image}"
-                            },
-                        },
-                        {"type": "text",
-                        "text": "Describe this image in detail please."
-                        },
+                        {"type": "text", "text": "What is in this image?"},
```

Comment:
Pull main and use the current iteration of the vision system prompt here, I think.